code/Model/RedirectorPageController.php

```diff
@@ -24,7 +24,12 @@ public function index(HTTPRequest $request)
         if (!$this->getResponse()->isFinished() && $link = $page->redirectionLink()) {
             $this->redirect($link, 301);
         }
-        return parent::handleAction($request, 'handleIndex');
+
+        $response = parent::handleAction($request, 'handleIndex');
```

Comment:
Note to myself that having a `parent::` call in the middle of a function isn't completely unheard of https://github.com/silverstripe/silverstripe-framework/blob/4/src/Security/Member.php#L1705

Reply:
That's a good example :)